src/main/java/org/assertj/core/util/introspection/ClassUtils.java

```diff
@@ -20,6 +20,57 @@
 public class ClassUtils {
 
   /**
+   * Determine whether the {@link Class} identified by the supplied name is present
+   * and can be loaded. Will return {@code false} if either the class or
+   * one of its dependencies is not present or cannot be loaded.
+   *
+   * @param className   the name of the class to check
+   * @param classLoader the class loader to use
+   *                    (may be {@code null}, which indicates the default class loader)
+   * @return whether the specified class is present
+   */
+  public static boolean isPresent(String className, ClassLoader classLoader) {
```

Comment:
should be removed as I don't see it used anywhere

Reply:
Same issue as the `protected`. I'll remove it